contracts/margined_insurance_fund/src/testing/tests.rs

```diff
@@ -902,93 +902,17 @@ fn test_pagination_limit() {
 
     instantiate(deps.as_mut(), mock_env(), info, msg).unwrap();
 
-    //add vamm
-    let addr1 = "addr0001".to_string();
-
-    let info = mock_info("addr0000", &[]);
-    let msg = ExecuteMsg::AddVamm { vamm: addr1 };
-
-    execute(deps.as_mut(), mock_env(), info, msg).unwrap();
-
-    //add second vamm
-    let addr2 = "addr0002".to_string();
-
-    let info = mock_info("addr0000", &[]);
-    let msg = ExecuteMsg::AddVamm { vamm: addr2 };
-
-    execute(deps.as_mut(), mock_env(), info, msg).unwrap();
+    //add eleven vamms
+    for n in 1..11 {
+        let mut addr = "addr000".to_string();
+        addr.push_str(&n.to_string());
```

Comment:
@maxrobot Its fine if  n < 10, but as soon as n = 10 then it inserts "addr00010" in the second slot (between "addr0001" and "addr0002")